examples/dm.rs

```diff
@@ -0,0 +1,102 @@
+use nostr::{gen_keys, util::nip04::decrypt, Event, Kind, Message};
+use std::{thread, time};
+use tungstenite::{connect, Message as WsMessage};
+use url::Url;
+
+const ALICE_SK: &str = "6b911fd37cdf5c81d4c0adb1ab7fa822ed253ab0ad9aa18d77257c88b29b718e";
+const BOB_SK: &str = "7b911fd37cdf5c81d4c0adb1ab7fa822ed253ab0ad9aa18d77257c88b29b718e";
+
+fn main() {
+    env_logger::init();
+
+    let (mut socket, _response) = connect(Url::parse("ws://localhost:3333/ws").unwrap())
+        .expect("Can't connect to Bob's relay");
+
+    let (_alice_keypair, alice_pubkey, alice_sk) = gen_keys(ALICE_SK);
+    let (_bob_keypair, bob_pubkey, bob_sk) = gen_keys(BOB_SK);
+
+    let alice_to_bob = "Hey bob this is alice (ping)";
+    let bob_to_alice = "Hey alice this is bob (pong)";
+
+    let alice_encrypted_msg =
+        Event::new_encrypted_direct_msg(alice_sk, &bob_pubkey, alice_to_bob.clone()).as_json();
+
+    let subscribe_to_alice = format!("sub-key:{}", alice_pubkey);
```

Comment:
Looking at Nostr specifications this is not the right way to make subscription. Is necessary build a JSON like `["REQ", <id>, <filter JSON>...]`. 

Related to filter, to sub for users messages is necessary use `authors` property of filter JSON. 

Complete Ex: `["REQ", "some_random_string", { "authors": ["alice_pk"] }]`

Reply:
fixed, don't know where I got this "sub-key" idea from